homeassistant/auth/providers/__init__.py

```diff
@@ -171,6 +171,7 @@ def __init__(self, auth_provider: AuthProvider) -> None:
         self._auth_manager = auth_provider.hass.auth  # type: ignore
         self.available_mfa_modules = {}  # type: Dict[str, str]
         self.created_at = dt_util.utcnow()
+        self.invalid_mfa_times = 0
```

Comment:
This too could be moved to a MFA flow if we had it, allowing each MFA implementation to decide it on their own.